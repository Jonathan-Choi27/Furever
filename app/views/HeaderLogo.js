import React, { Component } from "react";

import { View, Image, Header } from "react-native";
import Logo from "../../images/pet-search-logo.png";

export default class HeaderLogo extends Component {
  render() {
    return (
      // <View>
        <Image
          source={{
            uri:
              "https://firebasestorage.googleapis.com/v0/b/pet-search-soft3888.appspot.com/o/images%2Fpet-search-logo.png?alt=media&token=7b2f7152-027f-4b71-986d-35af8aac12b3",
          }}
          style={{
            flexDirection: "row",
            width: 100,
            height: 100,
          }}
        />      
      // </View>
    );
  }
}