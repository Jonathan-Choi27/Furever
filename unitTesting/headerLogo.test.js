// __tests__/Intro-test.js
import React from 'react';
import HeaderLogo from '../app/views/home/headerLogo';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<HeaderLogo/>).toJSON();
  expect(tree).toMatchSnapshot();
});