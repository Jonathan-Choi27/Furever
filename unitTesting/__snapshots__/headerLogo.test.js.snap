// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "uri": "https://firebasestorage.googleapis.com/v0/b/pet-search-soft3888.appspot.com/o/images%2FLogoBlack.png?alt=media&token=e1c8474a-de3f-46a6-ad62-f623d9f1f677",
    }
  }
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "height": 23,
      "width": 143,
    }
  }
/>
`;
